### ISSUE TITLE
How the `http_response_code:` works? 

### ISSUE THREAD

**Issue by rkrejcii** [getgrav/grav#1110]
I never used `http_response_code:` on our Grav websites.

However, actually I need some quick method how to temporary return `404` status code for promo page which expired, but might be used some time in the future with the same content.

So I was thinking that 

    http_response_code: 404

in the page's front-matter will do the job.

But nothing changed and I still see the content when I visit given page link.

The same when I tested above front-matter under Grav demo website.

So what I'm doing wrong please? 

Unfortunately, the Grav documentation doesn't provide too much info https://learn.getgrav.org/content/headers#http-response-code.

Thanks for any hint.

**Comment by flaviocopes** [comment]
This is the HTTP status code. I just tried and I got 

![screen shot 2016-10-17 at 15 25 21](https://cloud.githubusercontent.com/assets/11783388/19439284/dca89022-947d-11e6-9332-c22a2d31d7e8.png)

It *won't* trigger the web server 404 page but simply add the response code you specified to the browser, but the page *will* be shown. 

If you want to hide that page, set `published: false` in the header.
